client/components/clip-list-item.js

```diff
@@ -87,3 +92,21 @@ function findLabelTypeClassName(clipTitle) {
 		return classNames.SUPER
 	}
 }
+
+function formatDuration(frames, timeBase) {
+	const framesNumber = new Number(frames)
+	const timeBaseNumber = new Number(timeBase)
+
+	const frameNum = framesNumber % timeBaseNumber
+	const seconds = Math.floor(framesNumber / timeBaseNumber) % 60
+	const minutes = Math.floor(framesNumber / timeBaseNumber / 60) % 60
```

Comment:
Minor nitpick: `framesNumber / timeBaseNumber` could be done once, instead of thrice :)

Reply:
Good point!